cfbs.json

```diff
@@ -830,6 +830,20 @@
         "append enable.cf services/init.cf"
       ]
     },
+    "promise-type-scl-docker_compose": {
```

Comment:
```suggestion
    "promise-type-docker-compose": {
```

Reply:
I will drop it and use `docker_compose`

Reply:
Please use only dashes in the Build module name (and underscores in the promise type in policy).